scipy/linalg/src/id_dist/src/idz_frm.f

```diff
@@ -162,8 +162,8 @@ subroutine idz_permute(n,ind,x,y)
 c       y -- permutation of x
 c
         implicit none
-        integer n,k
-        complex*16 x(n),y(n),ind(n)
+        integer n,k,ind(n)
```

Comment:
I need to reverse this change (initially there was a warning for Complex to Integer Conversion) because of build failure. So, this type of warning stands as it is.

Reply:
Okay, let's leave this one as is then.